plain-text-feed/connector.py

```diff
@@ -0,0 +1,25 @@
+""" Copyright start
+  Copyright (C) 2008 - 2023 Fortinet Inc.
```

Comment:
Please update the copyright format with latest latest one.

Reply:
done